### DISCUSSION TITLE
错误: dial tcp 127.0.0.1:8080: connect: connection refused

### DISCUSSION THREAD

**Discussion by beiji-O-o** [Mrs4s/go-cqhttp#2215] [Q&A]
连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/onebot/v11/ws/ 时出现错误: dial tcp 127.0.0.1:8080: connect: connection refused

这个是为什么啊？

**Comment by zhx79513** [comment]
机器人没开，或者地址或端口没配置对
